roles/base/files/pacman.conf

```diff
@@ -1,9 +1,78 @@
+#
+# /etc/pacman.conf
```

Comment:
If we get rid of all the default comments we could make a cleaner diff between the file here and in master, which may make future maintenance easier.